pkg/cmd/mod/mod_push.go

```diff
@@ -127,7 +129,8 @@ func NewCmdPush(ioStreams genericiooptions.IOStreams) *cobra.Command {
 
 // AddFlags registers flags for a cli.
 func (flags *PushModFlags) AddFlags(cmd *cobra.Command) {
-	cmd.Flags().StringVarP(&flags.Version, "version", "v", flags.Version, "The version of the module e.g. '1.0.0' or '1.0.0-rc.1'.")
+	cmd.Flags().StringVarP(&flags.Version, "version", "v", "", "The version of the module e.g. '1.0.0' or '1.0.0-rc.1'.")
+	cmd.Flags().StringVar(&flags.OSArch, "osarch", "", "The os arch of the module e.g. 'darwin/arm64', 'linux/amd64'.")
```

Comment:
According to the `pushExample` above, here seems to be a typo? `osarch` ---> `os-arch`